scripts/zkai_attention_kv_stwo_fusion_mechanism_ablation_gate.py

```
@@ -0,0 +1,737 @@
+            handle.write(text)
+            handle.flush()
+            os.fsync(handle.fileno())
+        return tmp_path
+
+    try:
+        try:
+            for path, text in outputs:
+                original_bytes[path] = path.read_bytes() if path.exists() else None
+                tmp = write_temp(path, text)
+                temps.append(tmp)
+            for tmp, (path, _) in zip(temps, outputs):
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)